.storybook/preview-head.html

```diff
@@ -54,4 +54,10 @@
     --mb-color-bg-dashboard: var(--mb-color-bg-white);
     --mb-color-bg-dashboard-card: var(--mb-color-bg-white);
   }
+
+  /* For Embed frame questions to render properly */
+  #root:has([data-testid="embed-frame"]),
```

Comment:
Without height, stories won't be rendered properly when changing stories in the web view.

This was because if height is 0, chart won't be rendered
https://github.com/metabase/metabase/blob/ef7d2816840652472e76776b7be4025508ae9e50/frontend/src/metabase/visualizations/components/EChartsRenderer/ResponsiveEChartsRenderer.tsx#L35-L37